frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -666,6 +668,49 @@ export default function SurveyEdit(): JSX.Element {
                                                                 />
                                                             </div>
                                                         </LemonField.Pure>
+                                                        <LemonField.Pure
+                                                            label="Device Types"
+                                                            info="Select the device types to show the survey on"
+                                                        >
+                                                            <div className="flex flex-row gap-2 items-center">
+                                                                Device Types
+                                                                <LemonSelect
+                                                                    value={
+                                                                        value?.deviceTypesMatchType ||
+                                                                        SurveyMatchType.Contains
+                                                                    }
+                                                                    onChange={(matchTypeVal) => {
+                                                                        onChange({
+                                                                            ...value,
+                                                                            deviceTypesMatchType: matchTypeVal,
+                                                                        })
+                                                                    }}
+                                                                    data-attr="survey-device-types-matching-type"
+                                                                    options={Object.keys(SurveyMatchTypeLabels).map(
+                                                                        (key) => ({
+                                                                            label: SurveyMatchTypeLabels[key],
+                                                                            value: key,
+                                                                        })
+                                                                    )}
+                                                                />
+                                                                <PropertyValue
+                                                                    propertyKey="$device_type"
```

Comment:
can we use the constant from the taxonomy here?

Reply:
there's no easy constant for the `$device_type` key. I created a new function, `getPropertyKey` to return based on that now